src/utils/Utils.ts

```diff
@@ -475,11 +475,11 @@ export default class Utils {
   }
 
   public static computeSimplePrice(pricePerkWh: number, consumptionWh: number): number {
-    return Utils.truncTo(pricePerkWh * (consumptionWh / 1000), 6);
+    return Utils.roundTo(pricePerkWh * (consumptionWh / 1000), 6);
   }
 
   public static computeSimpleRoundedPrice(pricePerkWh: number, consumptionWh: number): number {
-    return Utils.truncTo(pricePerkWh * (consumptionWh / 1000), 2);
+    return Utils.roundTo(pricePerkWh * (consumptionWh / 1000), 2);
```

Comment:
Same here. The helper method shall be renamed by the way.

Reply:
The fix in OCPPUtils and in SimplePricingIntegration classes above should be sufficient to solve this issue.
Use of the truncTo() should not impact the total price of the transaction which is based on the computeSimplePrice() one.
You can revert to truncTo() after checking.

Reply:
@ellavas Did you get the last comments?

Reply:
I would prefer to remove this method and call the truncTo method on the computed price.

Reply:
The method was there because it is used somewhere else I think.